Mayuresh Jagtap/objects/objects.js

```diff
@@ -0,0 +1,227 @@
+// # Geometry Function Lab
+//
+// ### Part 1, Rectangle
+//
+// Given the following a `rectangle` object like the one below, write the following functions which accept a `rectangle` object as an argument:
+//
+// * isSquare - Returns whether the rectangle is a square or not
+// * area - Returns the area of the rectangle
+// * perimeter - Returns the perimeter of the rectangle
+//
+// ```javascript
+// const rectangleA = {
+//   length: 4,
+//   width: 4
+// };
+// ```
+//
+const rectangle = {
+  length: 4,
+  width: 4
+};
+
+const isSquare= function (){
+if (rectangle.length === rectangle.width){
+  console.log(`It is a square.`);
+} else {
+  console.log(`It is not a square.`);
+}
+}
+isSquare();
+
+  const area= function (){
+    const areaOfRectangle = rectangle.length * rectangle.width;
+    console.log(`The area for a rectangle is ${areaOfRectangle}.`);
+  return areaOfRectangle
+  }
+area( );
+
+const perimeter= function (){
+  const perimeterOfRectangle = 2 * (rectangle.length + rectangle.width);
+  console.log(`The perimeter for a rectangle is ${perimeterOfRectangle}.`);
+return perimeterOfRectangle
+}
+perimeter( );
+
+
+
+
+// ### Part 2, Triangle
+//
+// Given the following a `triangle` object like the one below, write the following functions which accept a `triangle` object as an argument:
+//
+// * isEquilateral - Returns whether the triangle is equilateral or not
+// * isIsosceles - Returns whether the triangle is isosceles or not
+// * area - Returns the area of the Triangle
+// * isObtuse - Returns whether the triangle is obtuse or not
+//
+// ```javascript
+// const triangleA = {
+//   sideA: 3,
+//   sideB: 4,
+//   sideC: 4
+// };
+// ```
+//
+const triangle = {
+   sideA: 3,
+   sideB: 4,
+   sideC: 4
+};
+
+const areaOfTri= function (){
```

Comment:
very minor nitpick that wont really effect the way the code works:
Instead of spacing your code out like this - 
`const areaOfTri= function (){`
You could try this instead, with the ()s up against the function?
`const areaOfTri = function(){`